src/Service/CheckGestion.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace App\Service;
+
+use App\Entity\ActionCheck;
+use App\Entity\RoadmapCheck;
+use App\Entity\StepCheck;
+use Doctrine\ORM\EntityManagerInterface;
+
+class CheckGestion
+{
+    public function checkAction(
+        ActionCheck $actionCheck,
+        EntityManagerInterface $entityManager
```

Comment:
No the entityManager should be injected in the construct of the service, not in the call of the function